pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-1335_osteo2_es_index.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This will not be enabled in this PR since it is not part of the master changelog.